README.md

```diff
@@ -28,27 +28,31 @@ In this task, Crypto-Metrics: This is a mobile web application used to check a l
 
 ## Interested in viewing the webpage:
 
-[Live Demo](https://rocket-mission-bookings.netlify.app/)
+[Live Demo](https://techemprez-crypto-metrics.netlify.app/)
 
 # Presentation Recording
 
-[View](paste link here)
+[View](https://www.loom.com/share/244b4c343b7348dc98d2d78b86e17bd0)
+
+---
 
 # Home Page
 
-The Rockets section displays a list of all available SpaceX rockets.
+- <img width="960" alt="image" src="https://user-images.githubusercontent.com/43815180/194651996-22cdfa03-f202-41ee-ba25-c5bf7ba1337c.png">
 
-![localhost 3000-SpaceTravelers'Hub](https://user-images.githubusercontent.com/83514256/193158547-3849a8f8-4832-4184-9a14-5aea419f54eb.png)
+---
 
 # Crypto Page
 
-The Missions section displays a list of current missions along with their brief description and participation status.
+- <img width="956" alt="image" src="https://user-images.githubusercontent.com/43815180/194652155-c85643f4-ecfe-4966-b494-010e9c36c1a3.png">
 
-![localhost 3000-SpaceTravelers'Hub(1)](https://user-images.githubusercontent.com/83514256/193158587-479909a4-2402-4aeb-9c57-ebe3acd61f13.png)
+---
 
 # Mobile view for home Page
 
-![localhost 3000-SpaceTravelers'Hub(2)](https://user-images.githubusercontent.com/83514256/193317548-f5efb2cd-2497-4e2d-b298-59a02d5c9ab9.png)
+- <img width="212" alt="image" src="https://user-images.githubusercontent.com/43815180/194652264-df2466a1-5d61-4484-b580-2aa9376fefe0.png">
+
+--
 
 ## Available Scripts
 
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " `The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.` ", please kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Done